src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSizeTests.cs

```diff
@@ -123,62 +110,6 @@ private string AddDifferenciatingSuffix(string filePath)
         return filePath;
     }
 
-    private void ScanForDifferences(Dictionary<string, long> tarEntries)
-    {
-        foreach (var entry in tarEntries)
-        {
-            if (!Baseline.TryGetValue(entry.Key, out long baselineBytes))
-            {
-                TrackDifference($"{entry.Key} does not exist in baseline. It is {entry.Value} bytes. Adding it to the baseline file.");
-                Baseline.Add(entry.Key, entry.Value); 
-            }
-            else
-            {
-                CompareFileSizes(entry.Key, entry.Value, baselineBytes);
-            }
-        }
-
-        foreach (var removedFile in Baseline.Keys.Except(tarEntries.Keys))
-        {
-            TrackDifference($"`{removedFile}` is no longer being produced. It was {Baseline[removedFile]} bytes.");
-            Baseline.Remove(removedFile);
-        }
-    }
-
-    private void CompareFileSizes(string filePath, long fileSize, long baselineSize)
-    {
-        // Only update the baseline with breaking differences. Non-breaking differences are file size changes
-        // less than the threshold percentage. This makes it easier to review the breaking changes and prevents
-        // inadvertently allowing small percentage changes to be accepted that can add up to a significant
-        // difference over time.
-        string breakingDifference = string.Empty;
-
-        if (fileSize == 0 && baselineSize != 0)
-        {
-            breakingDifference = $"'{filePath}' is now 0 bytes. It was {baselineSize} bytes.";
-        }
-        else if (fileSize != 0 && baselineSize == 0)
-        {
-            breakingDifference = $"'{filePath}' is no longer 0 bytes. It is now {fileSize} bytes.";
-        }
-        else if (baselineSize != 0 && (((fileSize - baselineSize) / (double)baselineSize) * 100) >= SizeThresholdPercentage)
-        {
-            breakingDifference =
-                $"'{filePath}' increased in size by more than {SizeThresholdPercentage}%. It was originally {baselineSize} bytes and is now {fileSize} bytes.";
-        }
-        else if (baselineSize != 0 && (((baselineSize - fileSize) / (double)baselineSize) * 100) >= SizeThresholdPercentage)
-        {
-            breakingDifference =
-                $"'{filePath}' decreased in size by more than {SizeThresholdPercentage}%. It was originally {baselineSize} bytes and is now {fileSize} bytes.";
-        }
-
-        if (!string.IsNullOrEmpty(breakingDifference))
-        {
-            TrackDifference(breakingDifference);
-            Baseline[filePath] = fileSize;
-        }
-    }
-
     private void TrackDifference(string difference) => Differences.AppendLine(difference);
 
     private void ReportDifferences()
```

Comment:
Is the WarnOnSdkContentDiffs check still applicable here?  This is no longer considered a Content Test.

Reply:
I feel like the functionality of the switch is still applicable. Maybe the switch should be `WarnOnDifferences` instead of `WarnOnSdkContentDiffs`? That way it can apply to all tests that report on differences?

Reply:
That makes sense. I'll remove the use of the switch in this test then. If the amount of breakage becomes an issue, we'll return to the drawing board on the implementation of this test rather than falling back onto a switch.